packages/light-client/dist/index.d.cts

```diff
@@ -0,0 +1,12 @@
+type DummyLightClient = {
+    /**
+     * The url being used to connect to the remote Tevm backend
+     */
+    url: string;
+    /**
+     * Name of the client
+     */
+    name: string;
+};
```

Comment:
Remove all the dist/ files

Reply:
done this through gitignore, should i totally prevent it from building through the package json or does this work ?